content/_includes/site/footer.njk

```diff
@@ -27,12 +27,40 @@
     </aside>
   {% endif %}
 
+  <div class="oddnews-signup">
```

Comment:
@sanajaved7 I moved this out of the `<footer role="contentinfo">` and then edited the grid to reflect the new structure I think that made sense then we don't have to worry about the footer specific link overrides.

Reply:
Perfect! That's what I had tried but didn't know how to update the grid, thanks so much!